pkg/resourcemanager/controller/garbagecollector/references/references.go

```diff
@@ -201,6 +201,10 @@ func computeAnnotations(spec corev1.PodSpec, additional ...string) map[string]st
 		}
 	}
 
+	for _, imagePullSecret := range spec.ImagePullSecrets {
```

Comment:
Can you also enhance the doc string of the `InjectAnnotations` function for this change?

Reply:
Done